share/ansible_navigator/utils/catalog_collections.py

```diff
@@ -64,9 +64,9 @@ def _catalog_plugins(self, collection: Dict) -> None:
         if file_manifest_file:
             file_path = f"{path}/{file_manifest_file}"
             if os.path.exists(file_path):
-                with open(file=file_path, encoding="utf-8") as read_file:
+                with open(file=file_path, encoding="utf-8") as fh:
```

Comment:
Should we drop the `file=` while we're at it?

Reply:
I'll get it when I pathlib join them all